cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -0,0 +1,50 @@
+% OCI(1) OCI-IMAGE-TOOL User Manuals
+% OCI Community
+% JULY 2016
+# NAME
+oci-image-tool-unpack \- Unpack an image or image source layout
+
+# SYNOPSIS
+**oci-image-tool unpack** [src] [dest] [flags]
+
+# DESCRIPTION
+`oci-image-tool unpack` unpacks a given OCI image into a directory suitable to be used with `runc`.
```

Comment:
On Fri, Jul 22, 2016 at 10:35:50AM -0700, W. Trevor King wrote:
>   Also translates the referenced config from
>   application/vnd.oci.image.serialization.config.v1+json to a
>   runtime-spec's config.json.

Probably worth working in DEST/config.json there somewhow.  Maybe:

  Also translates the referenced config from
  application/vnd.oci.image.serialization.config.v1+json to a
  runtime-spec-compatible DEST/config.json.